packages/typespec-azure-vscode/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
I guess we will also need 3rd part license file before release. @josefree , please help to confirm

Reply:
I am not sure if the license is with the repo. let me double check.

Reply:
Hi the answer is - 

Third-Party Attribution Notice: If your extension includes or depends on third-party libraries or assets, you are required to include a notice of third-party attribution. This ensures compliance with the licenses of those third-party components 
https://microsoft.sharepoint.com/:w:/t/VisualStudioMarketplace/ERStRuxo5wxGurt93n5lZP0BBYykB4aB56CjIXQpsSk50A?e=mlMA4n
![image](https://github.com/user-attachments/assets/a560ccd5-85d9-4bae-8773-e108e231e615)